### PR TITLE
[UI] Style and Help text changes

### PR DESCRIPTION
**Pull request by saminzadeh**
- Added help text in various locations
- Moved the create namespace form to top of the page

### CODE REVIEW

File: ui/src/components/RollupRuleEditor.js [hunk 1]
```diff
@@ -134,13 +144,16 @@ const TargetsEditorBase = props => {
                 <Icon type="delete" />
               </a>
             }>
-            <label>New Rollup Metric Name</label>
+            <label className="mb1">New Rollup Metric Name</label>
```
Comment (xichen2020) [inline ui/src/components/RollupRuleEditor.js]: Perhaps get rid of the `New` here?

File: ui/src/utils/helpText.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const helpText = {
+  namespace:
+    'Logical grouping of rules. This is often times your service name.',
```
Comment (xichen2020) [inline ui/src/utils/helpText.js]: Perhaps change "This is often times your service name." to "The namespace name must be a valid service name"?

Comment (saminzadeh) [reply ui/src/utils/helpText.js]: Done

File: ui/src/components/RollupRuleEditor.js (hunk 1 above)
Comment (saminzadeh) [reply ui/src/components/RollupRuleEditor.js]: Done

**Review by xichen2020 — APPROVED** [review]
LGTM

**Pull request merged**